index.html

```diff
@@ -4,48 +4,57 @@
     <meta charset="utf-8">
     <title>Mini Weather App</title>
     <link rel="stylesheet" href="css/style.css">
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"> -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+    <script src="js/main.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Chicle|Lato" rel="stylesheet">
   </head>
   <body>
     <main class="weather-app container">
 
-      <div class="row form-row">
-        <div class="col-md-4 col-md-offset-4">
+      <div class="header">
+        <div class="weat">
```

Comment:
Confusing class name... While it definitely works, it's not super clear to another dev.